mlibrary/pride
# Import `./src` into `./source` - Part 3

# Overview
With the code being split up and organized in the new `./src` directory, we are importing those new files into the current directory, and deleting the old files. This batch handles the remaining properties that have not been imported.

## Anything else?
`init` could not be directly added to the object. Instead, `Pride.init` had to be assigned to the import : `Pride.init = init;`.

## Testing
- Install the new list of dependencies (`npm install`).
- Build the repository (`npm run build`).
- Open `./SpecRunner.html` to see if the tests still pass.
- Go to your local Search repository and test the newly generated files in Search:
  1. Open `./package.json` and add `#import-src-into-source-part-3` at the end of the `pride` dependency URL:
  ```bash
  "pride": "git+https://github.com/mlibrary/pride.git#import-src-into-source-part-3"
   ``` 
  2. Do a clean install of Search:
  ```bash
  rm -rf node_modules && rm package-lock.json && npm install
   ``` 
  3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
